src/main/webapp/script.js

```diff
@@ -532,6 +580,75 @@ function highlightDiff(cy, mutList, reason = "") {
   };
 }
 
+/**
+ * Highlights the change in tokens of the given node based on the token mutation. If 
+ * tokens are added to the node, it adds them to the tippy in green. If tokens are deleted,
+ * it adds them to the tippy in red.
+ * @param node whose tokens have changed
+ * @param tokenMut the mutation made to the tokens of the node
+ */
+function indicateChangedTokens(node, tokenMut) {
+  const type = tokenMut["type_"] || -1;
+  const tokens = tokenMut["tokenName_"];
+
+  if (!type || !tokens || !node.tip) {
+    return;
+  }
+
+  // Get the div that contains the tooltip content
+  let tipContent = node.tip.popperChildren.content.firstChild;
+  switch (type) {
+    case 1:
+      // add tokens
+      // If this node already has an "Added Token" list, get it
+      let addedList = tipContent.querySelector(`#${node.data().id}-added`);
```

Comment:
Based on the bug we discussed via chat, I verified that doing `tipContent.querySelector(`#${CSS.escape(node.data().id)}-added`);` works here.

Same for the selector of `-deleted`

Reply:
Thank you :)